packages/frontend/src/components/wallet-migration/modals/CleanKeysModal/AccessKeyList.jsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from 'react';
+import { Translate } from 'react-localize-redux';
+import styled from 'styled-components';
+
+import classNames from '../../../../utils/classNames';
+import Accordion from '../../../common/Accordion';
+import Checkbox from '../../../common/Checkbox';
+import ChevronIcon from '../../../svg/ChevronIcon';
+import { ButtonsContainer, StyledButton } from '../../CommonComponents';
+
+const AccessKeyListContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+    text-align: left;
+
+    .access-key {
+        display: flex;
+        flex: 1;
+        font-weight: 600;
+        padding: 16px 0;
+        border-bottom: 1px solid lightgray;
+
+        .public-key {
+            flex: 7;
+
+            .key-prefix {
+                font-family: monospace;
+            }
+        }
+
+        .remove-checkbox {
+            flex: 1;
+        }
+    }
+
+    .account {
+        background-color: #fafafa;
+        border-bottom: 1px solid #f0f0f0;
+        display: flex;
+        flex: 1;
+        flex-direction: row;
+        width: 100%;
+
+        .account-detail {
+            flex: 7;
+
+            .account-id {
+                font-weight: bold;
+                text-overflow: ellipsis;
+            }
+
+            .keys-to-remove {
+                color: #de2e32;
+            }
+        }
+
+        .expand-keys {
+            flex: 1;
+            padding: 16px 0 0 0;
+        }
+    }
+`;
+
+const AccessKeyList = ({ account, onClose, onNext, selectKey, selectedKeys }) => {
+    const [expanded, setExpanded] = useState(true);
+    return (
+        <AccessKeyListContainer>
+            <h3 className='title'>
+                <Translate id='walletMigration.cleanKeys.accountTitle' />
+            </h3>
+            <div
+                className={classNames(['account', expanded ? 'open' : ''])}
+                id='full-access-keys'
+                onClick={() => setExpanded(!expanded)}
+            >
+                <div className='account-detail'>
+                    <p className='account-id'>{account.accountId}</p>
+                    <p className='keys-to-remove'>
+                        {account.accessKeys.length}/{account.totalAccessKeys}
```

Comment:
when I untick some accounts, this number doesn't get updated:

![image](https://user-images.githubusercontent.com/6027014/202340997-0078046f-05cd-49e7-bb0b-8ec39688a0bd.png)